bitnami/kubernetes-event-exporter/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: kubernetes-event-exporter
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kubernetes-event-exporter
   - https://github.com/resmoio/kubernetes-event-exporter
-version: 2.0.0
+version: 2.0.1
```

Comment:
You don't need to bump the version if it only affects the README.

Reply:
Didn't know that. Thank you!